HerPortal.BusinessLogic/Models/User.cs

```diff
@@ -8,4 +8,15 @@ public class User
 
     public List<LocalAuthority> LocalAuthorities { get; set; }
     public List<Consortium> Consortia { get; set; }
+
+    public List<string> GetAdministratedCustodianCodes()
+    {
+        var consortiumCodes = Consortia.Select(consortium => consortium.ConsortiumCode).ToList();
+        var custodianCodes = LocalAuthorities.Select(la => la.CustodianCode).ToList();
+        return LocalAuthorityData.LocalAuthorityConsortiumCodeByCustodianCode
+            .Where(codes => consortiumCodes.Contains(codes.Value))
+            .Select(codes => codes.Key)
```

Comment:
the logic here did confuse me a little initially, as the idea to
- iterate through every LA -> Consortium Pair
- filter to pairs referenced in the list
- get the key

for me it seems more natural to go the other way
- iterate through every consortium
- get the list of LAs for that consortium
- flatten them all together

which can be achieved by something like
```c#
consortiumCodes
  .SelectMany(consortiumCode => ConsortiumData.ConsortiumCustodianCodesIdsByConsortiumCode[consortiumCode])
  .Distinct(); // probably not necessary? an LA can only be in one consortium
```

Reply:
Yeah, this looks like a better solution!  I think I was looking at it from the wrong direction